code/style.css

```diff
@@ -1,4 +1,212 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/*GOOGLE FONT*/
```

Comment:
looks like the css file has the wrong indentation setting? Make sure that you're using 2 spaces indentation all through your code. (this is what we've decided in our code guidelines, at future work you might have another settings rule about this.)